modules/roxctl-generate-init-bundle.adoc

```diff
@@ -8,14 +8,14 @@
 [id="roxctl-generate-init-bundle_{context}"]
 = Generating an init bundle by using the roxctl CLI
 
-You can create an init bundle by using the the `roxctl` CLI.
+You can create an init bundle with secrets by using the `roxctl` CLI.
 
 .Prerequisites
-* You have configured the `ROX_API_TOKEN` and the `ROX_CENTRAL_ADDRESS` environment variables.
+* You have configured the `ROX_API_TOKEN` and the `ROX_CENTRAL_ADDRESS` environment variables. See xref:../cli/getting-started-cli.html#using-cli_cli-getting-started[Using the roxctl CLI] for more information.
```

Comment:
Unfortunately, we can't use xrefs within a module. It can be an "Additional resource" link at the assembly-level, however. Also, watch out for `.html` vs `.adoc` in the path.

Edit: Noticed a number of other instances throughout of xrefs in modules and the html vs adoc thing, so just apply this comment to those as well.